src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/SentenceKeyDates.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis
+
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.SentenceKeyDate
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.SentenceKeyDates
```

Comment:
Looks like a double import here

Reply:
Ahhhhhh, they're different classes. The idea is that `SentenceKeyDate` can be reused for the different key dates properties as they will all have a date, calculated date, override date etc.

![image](https://github.com/ministryofjustice/hmpps-integration-api/assets/42817036/3a8c39ad-b3a2-435f-84e3-d55df50e83f9)

![image](https://github.com/ministryofjustice/hmpps-integration-api/assets/42817036/a4125404-e861-45c5-8e71-1e99daec5bfd)